bats/fb-katello-container.bats

```diff
@@ -8,15 +8,25 @@ load foreman_helper
 load fixtures/content
 
 @test "try fetching container content" {
+  tContainerSlashesNotDefault
   tPackageExists podman || tPackageInstall podman
   podman login "${HOSTNAME}" -u admin -p changeme
   CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
   podman pull "${HOSTNAME}/${CONTAINER_PULL_LABEL}"
 }
 
+@test "try fetching container content with newer naming" {
+  tContainerSlashesDefault
+  tPackageExists podman || tPackageInstall podman
+  podman login "${HOSTNAME}" -u admin -p changeme
+  CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}/${PRODUCT_LABEL}/${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
```

Comment:
Instead of adding a new test, should we guard this variable behind a version check? Something like
```suggestion
  if tIsVersionNewer "${KATELLO_VERSION}" 4.16
    CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}/${PRODUCT_LABEL}/${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
  else
    CONTAINER_PULL_LABEL=$(echo "${ORGANIZATION_LABEL}-${PRODUCT_LABEL}-${CONTAINER_REPOSITORY_LABEL}"| tr '[:upper:]' '[:lower:]')
  fi
```

My reasoning is you always only run one or the other. If there was a version where you ran both tests then an additional test is obviously better.

Reply:
Sure, that works for me.